.github/workflows/cross-version-tests.yml

```diff
@@ -5,6 +5,12 @@ on:
     branches:
     - main
   pull_request:
+  workflow_dispatch:
+
+# https://stackoverflow.com/a/72408109/16358266
```

Comment:
this seems unrelated to the goals of this PR. can you tell me more about this change?

Reply:
Well, it allows you to run the tests manually, before opening a PR if needed.